.github/pull_request_template.md

```diff
@@ -1,23 +1,34 @@
 # Description
-<!-- Briefly describe the feature or bug that your pull request addresses -->
+<!-- Briefly describe the feature or bug that your pull request addresses, 1-2 sentences. -->
```

Comment:
Added reminder to keep it brief here, I always start writing more here before I remember the sections below